src/qubitmaps.jl

```diff
@@ -1,3 +1,4 @@
+using ITensors: ITensorMPS
```

Comment:
This is a bit of a nitpick, but I prefer the more verbose `using ITensors.ITensorMPS: ITensorMPS`, I think it makes it clearer what `ITensorMPS` is (i.e. that it is a submodule and not a type defined in `ITensors`, for example), and also makes it so that if you want to use a function or type from `ITensorMPS` you can add it on to the same `using` call, i.e. `using ITensors.ITensorMPS: ITensorMPS, siteinds`.

Reply:
Thanks for the explanation. I wasn't sure what you liked better about the more verbose version but those seem like good reasons.